tests/unit/test_loader_collection.py

```diff
@@ -429,6 +429,18 @@ def test_missing_readme(populated_collection_root):
     assert "README.md" == excinfo.value.missing_file
 
 
+@mock.patch("galaxy_importer.collection.CollectionLoader._build_docs_blob")
+def test_changelog_fail(_build_docs_blob, populated_collection_root, caplog):
+    _build_docs_blob.return_value = {}
+
+    data = CollectionLoader(
```

Comment:
```suggestion
    CollectionLoader(
```
I forgot to run `make lint` on my code changes, this should fix the one CI check. I think you'll need to rebase and sign to get other CI check to pass.